osquery/tables/system/darwin/shared_folders.mm

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <osquery/core.h>
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#import <OpenDirectory/OpenDirectory.h>
```

Comment:
Mind moving this up above the osquery includes? We typically roll with broadest to most specific scope for header includes.